pyproject.toml

```
@@ -1,3 +1,53 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "aiobotocore"
+version = "2.13.3"
+description = "Async client for aws services using botocore and aiohttp"
+readme = "README.rst"
+requires-python = ">=3.8"
+authors = [
+    { name = "Nikolay Novik", email = "nickolainovik@gmail.com" },
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Natural Language :: English',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Environment :: Web Environment',
+    'Framework :: AsyncIO',
+]
+
+dependencies = [
+    # NOTE: When updating botocore, always keep `project.optional-dependencies` aligned
+    "botocore>=1.34.70,<1.34.163",
+    "aiohttp>=3.9.2,<4.0.0",
+    "wrapt>=1.10.10, <2.0.0",
+    "aioitertools>=0.5.1,<1.0.0",
```

I see a whitespace in the middle. Why not just make the style consistent?
```suggestion
    "botocore >=1.34.70, <1.34.163",
    "aiohttp >=3.9.2, <4.0.0",
    "wrapt >=1.10.10, <2.0.0",
    "aioitertools >=0.5.1, <1.0.0",
```